torch_xla/csrc/runtime/computation_client.h

```diff
@@ -30,6 +30,9 @@
 namespace torch_xla {
 namespace runtime {
 
+// Forward declaration
+class XlaCoordinator;
```

Comment:
Why do you need the forward declaration here? Can you `#include` `XlaCoordinator` instead?

Reply:
I tried that, but since XlaCoordinator includes a pjrt header, it redefined a ton of log macros and broke the build...

Reply:
Added a comment and removed the redundant forward declaration in `xla_coordinator.h`